### ISSUE TITLE
Use proton environment when using winetricks

### ISSUE THREAD

**Issue by Lucki** [tkashkin/GameHub#242]
Currently the system wine version gets used for winetricks even if proton is selected for the current prefix. This results in a upgrade of that said prefix to the system wine version. After making your changes and starting the game again with proton the prefix gets downgraded to the selected proton version.
This results in unnecessary time waiting for the upgrade and downgrade, unnecessary write cycles (SSDs anyone?) and maybe needed user intervention if wine-mono isn't installed in the system (popup asking if I want to install it).

This can be prevented if the proton environment for such task is being used.
I'm not sure which environment variables are mandatory to set but you probably already know them because winecfg, taskmgr (and kill?) are already set correctly:
~~~
echo "$WINE, $WINEDLLPATH, $WINELOADER, $WINESERVER, $WINETRICKS"
/mnt/steam/windows/steamapps/common/Proton 4.2/dist/bin/wine, /mnt/steam/windows/steamapps/common/Proton 4.2/dist/lib64/wine:/mnt/steam/windows/steamapps/common/Proton 4.2/dist/lib/wine, /mnt/steam/windows/steamapps/common/Proton 4.2/dist/bin/wine, /mnt/steam/windows/steamapps/common/Proton 4.2/dist/bin/wineserver, /usr/bin/winetricks
~~~

**Comment by tkashkin** [comment]
Should be fixed in 202e4d2. Let me know if there's more variables to override.

**Comment by Lucki** [comment]
Thanks! Looks like it's picking up the proton environment now.
